src/providers/info.rs

```diff
@@ -76,6 +77,7 @@ struct Manifest {
 pub struct TokenInfo {
     #[serde(rename = "type")]
     pub token_type: TokenType,
+    // No need to map to AddressEx as the information are present in this struct
```

Comment:
This would also break deserialisation as we rely on the same structure for the backend returned types.